react-charts/area-chart/intro.md

```diff
@@ -1 +1,15 @@
-In this scenario, you will learn how to use PatternFly React area chart components for the first time.
+In this scenario, you will learn how to use PatternFly React area chart components.
+
+You will begin by adding a CSS selector to define the chart size...
```

Comment:
Let's cut out the "You will" here and instead kick off with "Begin by adding.." This way, we use less words and get to the point quicker. Also, we're more aligned with the Technical Writing Style Guide on Mojo that advises we stick with present tense and avoid future tense.